tls.go

```diff
@@ -137,9 +138,21 @@ func (t *TLSFlags) GetTLSConfig() (*tls.Config, error) {
 	}
 
 	if t.CipherSuite != "" {
-		ret.CipherSuites = cipherMap[t.CipherSuite]
-		if ret.CipherSuites == nil {
-			return nil, fmt.Errorf("%s is not a valid cipher suite", t.CipherSuite)
+		// allow either one of our standard values (e.g., chrome) or a comma-delimited list of ciphers
+		if _, ok := cipherMap[t.CipherSuite]; ok {
+			ret.CipherSuites = cipherMap[t.CipherSuite]
+		} else {
+			strCiphers := getCSV(t.CipherSuite)
+			var intCiphers = make([]uint16, len(strCiphers))
+			for i, s := range strCiphers {
+				s = strings.TrimPrefix(s, "0x")
+				v64, err := strconv.ParseUint(s, 16, 16)
```

Comment:
Here you *could* use radix = 0 to interpret the 0x automatically (and if they decide they want to put it in decimal or octal, that would work too). But then, that may not be desirable in this case.

Either way, this should probably just go into zflags -- []typeX just runs the typeX parser on each comma-separated element.